### PR TITLE
Reminds you about the blackout policy when you pop out of cloner.

### PR DESCRIPTION
**Pull request by Azarak**
## About The Pull Request
Just a reminder of our policy.

## Why It's Good For The Game
We have many and it's not too hard to forget it, reminder is good

## Changelog
:cl:
add: Reminds you about blackout policy when you pop out of cloning
/:cl:

### CODE REVIEW

File: code/game/machinery/cloning.dm [hunk 1]
```diff
@@ -379,6 +379,7 @@
 
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		mob_occupant.grab_ghost()
+		to_chat(occupant, "<span class='warning'><b>You remember nothing after you've blacked out and you do not remember who or what events killed you, however, you can have faint recollection of what led up to it.</b>") //Skyrat change - reminds you about the blackout policy
```
Comment (Akrilla) [inline code/game/machinery/cloning.dm:382]: Maybe even worth a bigger text class on here?

Comment (Azarak) [reply code/game/machinery/cloning.dm:382]: It's <b>old though

Comment (Akrilla) [reply code/game/machinery/cloning.dm:382]: yeah I guess, prob fine

**Pull request merged**